client.js

```diff
@@ -62,8 +62,25 @@ class Client {
     if (errors || !data) throw new Error(formatJSON({ data, errors }));
     return { ...data };
   }
+
+  async updateFlow(flowId, liveFlowData) {
+    const { data, errors } = await this.graphQL(
+      `mutation UpdateFlow ($flowId: uuid!, $liveData: jsonb!) {
+        update_flows_by_pk(pk_columns: {id: $flowId}, _set: {data: $liveData}) {
+          id
+        }
+      }`,
+      {
+        flowId: flowId,
+        liveData: liveFlowData,
+      }
+    );
+    if (errors || !data) throw new Error(formatJSON({ data, errors }));
+    return { ...data };
+  }
```

Comment:
many `-files` flows used as external portals aren't published and we don't necessarily want to publish them before running this script - so now it handles the update mutation conditionally depending if a flow is published or not!